config/country_dialing_codes.yml

```diff
@@ -774,6 +890,22 @@ UZ:
   name: Uzbekistan
   country_code: '998'
   sms_only: true
+VA:
+  name: Vatican City (Città del Vaticano)
+  country_code: '39'
+  sms_only: true
+VC:
+  name: Saint Vincent and the Grenadines
+  country_code: '1784'
+  sms_only: true
+VG:
+  name: Virgin Islands (British)
+  country_code: '1284'
+  sms_only: true
+VI:
+  name: Virgin Islands (U.S.)
+  country_code: '1340'
+  sms_only: false
```

Comment:
Thanks for adding these countries. Could you please alphabetize them by country name as opposed to 2-letter code? They are displayed alphabetically by name in the dropdown.

Reply:
On it!

Reply:
Updated 👍